pkg/kube/util/names/names.go

```diff
@@ -128,21 +129,24 @@ func GetVersionFromVersionedSecretName(name string) (int, error) {
 // JobName returns a unique, short name for a given eJob, pod(if exists) combination
 // k8s allows 63 chars, but the pod will have -\d{6} appended
 // IDEA: maybe use pod.Uid instead of rand
```

Comment:
Since you implemented this, please remove the comment.

Reply:
Removed